addons/web_editor/static/lib/odoo-editor/src/utils/utils.js

```diff
@@ -1831,10 +1850,10 @@ export function setTagName(el, newTagName) {
     while (el.firstChild) {
         n.append(el.firstChild);
     }
-    const closestLi = el.closest('li');
+    const closestLi = closestElement(el, 'div, li');
```

Comment:
This looks like, this case is specifically handling scenario for snippets in website is it? Can you write test case for such that this code be justified ! Also the variable name doesnot justify the value its holding because the [line](https://github.com/odoo/odoo/pull/117304/files#diff-31fc67a4cad2b2dd27492e4d94f31c1bf110d27d30f27a3ab19b7cb113d28479R1856) is not making sense to me. Try to find a better one for it.